array_api_tests/test_data_type_functions.py

```diff
@@ -174,6 +175,33 @@ def test_iinfo(dtype):
     # TODO: test values
 
 
+if api_version >= "2022.12":
```

Comment:
Yeah, doesn't feel good. I (seemingly) have to do this for a test like `test_conj` because we're using the `>=2022.12` feature of Hypothesis `xps.complex_dtypes()`,

https://github.com/data-apis/array-api-tests/blob/a533680532bf79cfd6aebbe65a202dc458e7ecfd/array_api_tests/test_operators_and_elementwise_functions.py#L927-L934

so kinda just did this to be consistent with the pattern. But I think the ultimate goal with `test_conj`/etc. would be to not use this pattern, so now skipping this test instead in my force-push.